specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/account.json

```diff
@@ -0,0 +1,954 @@
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AutomationAccount": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AutomationAccountProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the automation account properties."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Gets or sets the etag of the resource."
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "Resource system metadata."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../common/v1/definitions.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Definition of the automation account type."
+    },
+    "AutomationAccountProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets the SKU of account."
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "description": "Gets or sets the last modified by."
+        },
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets status of account.",
+          "enum": [
+            "Ok",
+            "Unavailable",
+            "Suspended"
+          ],
+          "x-ms-enum": {
+            "name": "AutomationAccountState",
+            "modelAsString": true
+          }
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Gets the creation time.",
+          "x-nullable": false
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Gets the last modified time.",
+          "x-nullable": false
+        },
+        "description": {
+          "type": "string",
+          "description": "Gets or sets the description."
+        },
+        "encryption": {
+          "$ref": "#/definitions/EncryptionProperties",
+          "description": "Encryption properties for the automation account"
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "items": {
+            "$ref": "../../common/v1/definitions.json#/definitions/PrivateEndpointConnection"
+          },
+          "description": "List of Automation operations supported by the Automation resource provider."
+        },
+        "publicNetworkAccess": {
+          "type": "boolean",
+          "description": "Indicates whether traffic on the non-ARM endpoint (Webhook/Agent) is allowed from the public internet"
+        },
+        "disableLocalAuth": {
```

Comment:
Enum should be preferred over boolean properties. Consider making this property name localAuth with values Enabled/Disabled. 
Refer best practice 'Replace boolean/switch properties with better enum' in https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html

Reply:
All other microservices are developed expecting this value to be boolean and it is expected a boolean for policy

Reply:
Given the API already implemented and you have to meet Vanguard deadline, I am granting exception on this. 

For new properties that will be added in future, implement as enum instead of boolean.